backend/FwLite/FwDataMiniLcmBridge/FwDataBridgeConfig.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace FwDataMiniLcmBridge;
+
+public class FwDataBridgeConfig
+{
+    public string ProjectsFolder { get; set; } = @"C:\ProgramData\SIL\FieldWorks\Projects";
+    public string TemplatesFolder { get; set; } = @"C:\Program Files\SIL\FieldWorks 9\Templates";
```

Comment:
On Linux these will need to be something involving the XDG_DATA_DIR environment variable (similarly to https://github.com/sillsdev/languageforge-lexbox/issues/1005).

Reply:
good catch, yeah this branch was behind, so I've merged develop into it and fixed this code now.